index.html

```diff
@@ -1,52 +1,27 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf-8">
-        <title>Anidata Starters</title>
+---
+layout: default
```

Comment:
Yeah I'm open to writing a new layout! I think it makes sense so that we aren't loading unnecessary dependencies on other pages in the site